src/app/design/styles/spacing/index.html

```diff
@@ -1,61 +1,287 @@
+					<li><a routerLink="/components/card">Card</a></li>
+					<li><a routerLink="/components/popover">Popover</a></li>
+					<li><a routerLink="/components/repeater">Repeater</a></li>
+        </ul>
+      </div>
+      <div class="definition-code">
+        <div class="sky-field-label">Code</div>
+        <div><stache-code>class="sky-padding-even-default"</stache-code></div>
+      </div>
+      <div class="definition-styling">
+        <div class="sky-field-label">Styling</div>
+				<div>padding: 10px</div>
+      </div>
+    </div>
+	</div>
+</div>
+	
+<stache-page-anchor>Content</stache-page-anchor>
+<div style="margin-bottom: 20px;">
+	<div class="sky-subsection-heading">Vertically-arranged content</div>
+	
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="sky-subsection-heading" style="background-color: #c1e8fb; width: 50px; height: 25px; margin-bottom: 5px;"></div>
+			<div class="sky-subsection-heading" style="background-color: #c1e8fb; width: 50px; height: 25px;"></div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="sky-field-label">Usage</div>
+        <div>Closely-related elements have a small amount of margin between them so that they stay visually linked. Examples include:</div>
+        <ul>
+          <li><a routerLink="/components/form" fragment="control-label">Form field labels</a></li>
+					<li>Section header and the first element in that section</li>
+        </ul>
+      </div>
+      <div class="definition-code">
+        <div class="sky-field-label">Code</div>
+        <div><stache-code>class="sky-margin-stacked-compact"</stache-code></div>
+      </div>
+      <div class="definition-styling">
+        <div class="sky-field-label">Styling</div>
+				<div>margin-bottom: 5px</div>
+      </div>
+    </div>
+	</div>
+	
+	<div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="sky-subsection-heading" style="background-color: #c1e8fb; width: 50px; height: 25px; margin-bottom: 10px;"></div>
+			<div class="sky-subsection-heading" style="background-color: #c1e8fb; width: 50px; height: 25px;"></div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="sky-field-label">Usage</div>
+        <div>Multiple instances of a type of object need enough vertical spacing so that the objects can be distinguished from each other, but not so much that they seem unrelated. Examples include:</div>
+        <ul>
+          <li>Form inputs</li>
+        </ul>
+      </div>
+      <div class="definition-code">
+        <div class="sky-field-label">Code</div>
+        <div><stache-code>class="sky-margin-stacked-default"</stache-code></div>
+      </div>
+      <div class="definition-styling">
+        <div class="sky-field-label">Styling</div>
+				<div>margin-bottom: 10px</div>
```

Comment:
We have not been using code snippet for these, since they technically use CSS notation they are meant to be generally applicable/usable on whatever platform.